packages/core/src/components/group-item/group-item.tsx

```diff
@@ -60,18 +60,15 @@ export class GroupItem {
   /**
    * Selection changed
    */
-  @Event() selectedChanged: EventEmitter<HTMLIxGroupItemElement>;
+  @Event() selectedChanged?: EventEmitter<HTMLIxGroupItemElement>;
 
   /**
    * Index
    */
-  @Prop() index: number;
+  @Prop() index?: number;
 
   @Listen('click', { passive: true })
   clickListen() {
-    if (this.suppressSelection) {
-      return;
-    }
```

Comment:
Removing this will start trigger event handler on parent. This should no happen if suppressSelection is set

Reply:
added corresponding logic inside the parent